app/status/views.py

```diff
@@ -1,9 +1,46 @@
-from flask import jsonify
+from flask import jsonify, current_app
 
 from . import status
 from . import utils
+from ..main.helpers.service import ServiceLoader
+from ..main import main
 
 
 @status.route('/_status')
 def status():
-    return jsonify(status="ok", version=utils.get_version_label())
+
+    # ServiceLoader is the only thing that actually connects to the API
+    service_loader = ServiceLoader(
```

Comment:
ServiceLoader is going away in #43, maybe it's worth rebasing on top of that branch. This also means that we need a new place for the `status` method, which also exists in supplier-frontend `api_client` and as `get_api_status` in buyer-frontend `models.py`. I think it's worth replacing all of them with a `dmutils` base client method right away